[2022-02-09 09:01:17,236] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-09-01T23:00:00+00:00 [queued]>
[2022-02-09 09:01:17,255] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-09-01T23:00:00+00:00 [queued]>
[2022-02-09 09:01:17,256] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:01:17,256] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 09:01:17,256] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:01:17,277] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2021-09-01 23:00:00+00:00
[2022-02-09 09:01:17,289] {standard_task_runner.py:52} INFO - Started process 532 to run task
[2022-02-09 09:01:17,302] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'local_to_gcs_task', 'scheduled__2021-09-01T23:00:00+00:00', '--job-id', '511', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingestee_gcs_dag.py', '--cfg-path', '/tmp/tmpc0fu_qj3', '--error-file', '/tmp/tmpvoz4qk10']
[2022-02-09 09:01:17,304] {standard_task_runner.py:77} INFO - Job 511: Subtask local_to_gcs_task
[2022-02-09 09:01:17,410] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-09-01T23:00:00+00:00 [running]> on host cc021fb126ed
[2022-02-09 09:01:17,484] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 09:01:17,523] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2021-09-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-01T23:00:00+00:00
[2022-02-09 09:01:18,092] {python.py:175} INFO - Done. Returned value was: None
[2022-02-09 09:01:18,108] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=local_to_gcs_task, execution_date=20210901T230000, start_date=20220209T090117, end_date=20220209T090118
[2022-02-09 09:01:18,173] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 09:01:18,233] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-09 09:17:05,429] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-09-01T23:00:00+00:00 [queued]>
[2022-02-09 09:17:05,447] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-09-01T23:00:00+00:00 [queued]>
[2022-02-09 09:17:05,447] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:17:05,448] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 09:17:05,448] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:17:05,467] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2021-09-01 23:00:00+00:00
[2022-02-09 09:17:05,476] {standard_task_runner.py:52} INFO - Started process 701 to run task
[2022-02-09 09:17:05,490] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'local_to_gcs_task', 'scheduled__2021-09-01T23:00:00+00:00', '--job-id', '616', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingesteee_gcs_dag.py', '--cfg-path', '/tmp/tmp5fllnwtt', '--error-file', '/tmp/tmp9qvrnl5l']
[2022-02-09 09:17:05,492] {standard_task_runner.py:77} INFO - Job 616: Subtask local_to_gcs_task
[2022-02-09 09:17:05,600] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-09-01T23:00:00+00:00 [running]> on host 75953ff4f3d5
[2022-02-09 09:17:05,681] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 09:17:05,746] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2021-09-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-01T23:00:00+00:00
[2022-02-09 09:17:06,375] {python.py:175} INFO - Done. Returned value was: None
[2022-02-09 09:17:06,394] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=local_to_gcs_task, execution_date=20210901T230000, start_date=20220209T091705, end_date=20220209T091706
[2022-02-09 09:17:06,436] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 09:17:06,478] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
